beta/project-griffon/using-project-griffon/custom-validation-and-project-griffon.md

```diff
@@ -0,0 +1,61 @@
+# Custom Validation and Project Griffon
+## Overview
+The Custom Validation view provides a quick and easy solution to write your own JavaScript function to validate the events in your session.
+
+## Using the Custom Validation View
+
+Follow these steps to get started:
+
+1. Ensure you have implemented the latest version of [Project Griffon](../set-up-project-griffon.md) extension
+2. Visit https://experience.adobe.com/griffon \(or griffon.adobe.com\).
+3. Connect your app to a Project Griffon session
+4. Select the subview Custom Validation \(per the screenshot below\) to view the code editor.
+
+![](../../../.gitbook/assets/griffon-custom-validation-navigation.png)
+
+### Writing a Validation Function
+
+The function should expect an array of Project Griffon events and should return a boolean (true or false) or an object that contains a status property and optionally an array of event uuids.
```

Comment:
I'm fine with adding a fifth step but I think there's enough content in writing a validation function that it warrants some subheader. All the content that proceeds past here is relevant to understanding how to write a validation function. And users can potentially run the validation script without needing to write one

Reply:
Sorry, if I wasn't clear...I don't mean add a 5th step. I meant that we should 1 remove the subheading and 2) include this sentence under the procedure as an "Important" note. This way, users won't miss the information if they need it. It seems odd to have a subheading with just 1 sentence under it.

Reply:
Hi @dgrossen, it looks like my change didn't get made...let me know if you want to discuss it.  What I was asking for is this:

- Remove the heading

- Put the text into an "Important" note like this:
"**Important**: The function should expect an array of Project Griffon events and should return a boolean (true or false) or an object that contains a status property and optionally an array of event uuids."

This text seems related to the procedure, and it's only 1 sentence, so it should not in a separate section with  a heading. Thanks!

Reply:
I didn't change it because I am suggesting that everything following the subheading pertains to writing a validation function. In order to write a validation function one has to be familiar with the event definitions ("Project Griffon event definition" and the "Project Griffon annotation definition"), how to return a valid result ("Returning validation results"), saving and deleting their functions ("Validation Script Management"), and troubleshooting. It's all within the context of writing a function

Reply:
Hi @dgrossen, I don't agree with the reasoning as I think the information will have the same importance & context as an "Important" note. But, no problem...I'll go ahead and merge the PR.

Reply:
Looks like you might have not made the other changes, so I'll just resolve this conversation.